more/free-programming-cheatsheets.md

```diff
@@ -116,6 +116,8 @@
 ### Data Science
 
 * [Cheatsheets for Data Scientists](https://www.datacamp.com/community/data-science-cheatsheets) - Datacamp (PDF)
+* [Data Science Cheat Sheets](https://medium.com/analytics-vidhya/data-science-cheat-sheets-109ddcb1aca8) - Surya Gutta (HTML, PDF)
```

Comment:
This is a blog post that embeds 3 cheatsheets from other sites. We prefer to link directly (and give credit to) to the original sources.